pkg/pipelineascode/pipelineascode.go

```diff
@@ -166,7 +167,13 @@ func (p *PacRun) startPR(ctx context.Context, match matcher.Match) (*tektonv1.Pi
 		}
 
 		if err = p.k8int.CreateSecret(ctx, match.Repo.GetNamespace(), authSecret); err != nil {
-			return nil, fmt.Errorf("creating basic auth secret: %s has failed: %w ", authSecret.GetName(), err)
+			if errors.IsAlreadyExists(err) {
```

Comment:
Please add a comment above explaining the case we might end up with an existing secret based on what we've investigated so far. Maybe also mention that this is a workaround for the issue, rather than designed behavior. Other than that the change looks good to me.

Reply:
https://github.com/openshift-pipelines/pipelines-as-code/issues/2263